CHANGELOG.md

```diff
@@ -8,7 +8,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.0] - 2022-08-22
+
+### Breaking
```

Comment:
Keep a Changelog (which this changelog claims to be formatted after up top) uses "changed"

```suggestion
### Changed
```